File: Desafios em C#/1131 - Grenais.cs
using System; 

//Developed by: @LucasMarcuzo
class URI1131 {
    static void Main(string[] args) { 

       int x, y, i, g, e, s, grenais;
       x=y=i=g=e=s=grenais=0;

       do
       {
            string[] array = Console.ReadLine().Split(' ');

            x=int.Parse(array[0]);
            y=int.Parse(array[1]);

           if(x > y)
           {
               i++;
               grenais++;
            
           }else if(y > x)
           {
               g++;
               grenais++;
           }else
           {
               e++;
               grenais++;
           }

           Console.WriteLine("Novo grenal (1-sim 2-nao)");
           s=int.Parse(Console.ReadLine());
           
       } while (s == 1);


       if (i > g)
       {
           Console.WriteLine("{0} grenais",grenais);
           Console.WriteLine("Inter:{0}",i);
           Console.WriteLine("Gremio:{0}",g);
           Console.WriteLine("Empates:{0}",e);
           Console.WriteLine("Inter venceu mais");

       }else if (g > i)
       {
           Console.WriteLine("{0} grenais",grenais);
           Console.WriteLine("Inter:{0}",i);
           Console.WriteLine("Gremio:{0}",g);
           Console.WriteLine("Empates:{0}",e);
           Console.WriteLine("Gremio venceu mais");

       }else
       {
           Console.WriteLine("{0} grenais",grenais);
           Console.WriteLine("Inter:{0}",i);
           Console.WriteLine("Gremio:{0}",g);
           Console.WriteLine("Empates:{0}",e);
           Console.WriteLine("Nao houve vencedor");
       }

    }
}

File: Desafios em JavaScript/1131 - Grenais.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

var x, y, i, g, e, s, grenais;
x=y=i=g=e=s=grenais=0;

do
{
    let ln = lines.shift().split(` `);
    x=parseInt(ln.shift());
    y=parseInt(ln.shift());

    if(x > y)
    {
        i++;
        grenais++;
        
    }else if(y > x)
    {
        g++;
        grenais++;
    }else
    {
        e++;
        grenais++;
    }

    console.log(`Novo grenal (1-sim 2-nao)`);
    s=parseInt(lines.shift());
    
} while (s === 1);


if (i > g)
{
    console.log(`${grenais} grenais`);
    console.log(`Inter:${i}`);
    console.log(`Gremio:${g}`);
    console.log(`Empates:${e}`);
    console.log(`Inter venceu mais`);

}else if (g > i)
{
    console.log(`${grenais} grenais`);
    console.log(`Inter:${i}`);
    console.log(`Gremio:${g}`);
    console.log(`Empates:${e}`);
    console.log(`Gremio venceu mais`);

}else
{
    console.log(`${grenais} grenais`);
    console.log(`Inter:${i}`);
    console.log(`Gremio:${g}`);
    console.log(`Empates:${e}`);
    console.log(`Nao houve vencedor`);
}
